src/index.css

```diff
@@ -1,27 +1,34 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Open Sans', sans-serif;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+.calc-grid {
+  display: grid;
+  grid-template-columns: repeat(4, 5rem);
+  height: 50vh;
+  justify-content: center;
+  margin-top: 8rem;
+}
+
+.zero {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.operator {
+  background-color: rgb(214, 140, 82);
+}
```

Comment:
- [ ] Please make sure to use the correct color for the operators' buttons background `#FFA600` to follow the wireframe design.